Luigi 2.7.0 or later are not supported

toru-takahashi opened the pull request:
Luigi 2.7.0 removedluigi.{postgres, s3}, introduced luigi.contrib.{..} instead.
https://github.com/spotify/luigi/releases/tag/2.7.0

Then, luigi-td doesn't work on Luigi 2.7.0 for now.

toru-takahashi committed:
Luigi 2.7.0 or later are not supported

Luigi 2.7.0 removedluigi.{postgres, s3}, introduced luigi.contrib.{..} instead.
https://github.com/spotify/luigi/releases/tag/2.7.0

Then, luigi-td doesn't work on Luigi 2.7.0 for now.

yyuu commented on requirements.txt:
```diff
@@ -1,5 +1,5 @@
 six>=1.9.0
 boto>=2.36.0
-luigi>=1.1.2
+luigi>=1.1.2,<2.7.0
```
@toru-takahashi this package is actually relying on `luigi < 2.5.0`. I confirmed that test passing with `luigi==2.4.0`

toru-takahashi replied:
Thank you. I'll fix it later.

toru-takahashi committed:
Fix the version

Ref. https://github.com/treasure-data/luigi-td/pull/5
> @toru-takahashi this package is actually relying on luigi < 2.5.0. I confirmed that test passing with luigi==2.4.0

toru-takahashi commented:
@yyuu  Fixed!
